README.md

```diff
@@ -1,2 +1,164 @@
 # github-backup
 :back: :arrow_up: A script to backup your GitHub Account/Organisation so you can still work when (they/you are) offline.
+
+## Why?
+
+As a person who uses GitHub as their "_single-source-of-truth_"
```

Comment:
this is _one_ of the _valid_ use-cases. 👍 
the _other_ is: 
We use "checklists" in GitHub issues as the "_acceptance criteria_".
to avoid ["scope creep"](https://en.wikipedia.org/wiki/Scope_creep) i.e. "_moving goalposts_",
(_where people add items to the checklist without accountability_) 
we need to _track_ the changes made to the issue description/comments.